werkzeug/_internal.py

```diff
@@ -282,8 +284,8 @@ def _cookie_parse_impl(b):
         if not match:
             break
 
-        key = match.group('key').strip()
-        value = match.group('val')
+        key = (match.group('key') or '').strip()
+        value = match.group('val').strip()
```

Comment:
I don't think this is correct, I think it should be `lstrip` because otherwise we'd break spaces in values.